src/home_robot/home_robot/datasets/spot/data_utils.py

```diff
@@ -0,0 +1,342 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+import argparse
+import dataclasses
+import datetime
+import pickle as pkl
+import sys
+import timeit
+from pathlib import Path
+from typing import Optional, Sequence, Tuple, Union
+
+import bosdyn.client.frame_helpers as frame_helpers
```

Comment:
we probably want this in home_robot_spot not in home_robot proper?
I guess it will only be lazily imported for now, but can you file an issue for when we merge spot code?